Documentation~/Object_ProBuilderize.md

```diff
@@ -1,15 +1,23 @@
-# ![ProBuilderize icon](images/icons/Object_ProBuilderize.png) ProBuilderize
+# ProBuilderize a standard GameObject
 
-The __ProBuilderize__ action converts the selected object(s) into ProBuilder-editable objects.
+You can turn any 3D GameObject to a ProBuilder object. 
 
-From the main menu, select **Tools** > **ProBuilder** > **Object** > **ProBuilderize**.
+To ProBuilderize a GameObject:
+
+1. In the **Scene** view, select the GameObject.
+2. Do one of the following:
+    * From the main menu, select **Tools** > **ProBuilder** > **Object** > **Pro Builderize**.
+    * Right-click the GameObject and select **MeshFilter** > **ProBuilderize**.
+3. The **ProBuilderize** overlay opens, and you can customize the ProBuilder object.
 
 ## ProBuilderize options
 
-Use the following ProBuilderize options to customize how your object is converted to a ProBuilder object.
+Use the options in the **ProBuilderize** overlay to customize the ProBuilder object.
+
+| **Property** | **Description** |
+| :--- | :--- |
+| **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles where possible.<!--can we give an example of where it won't be possible?--> |
```

Comment:
You didn't misunderstand the question :)
Out of curiosity, why would people choose quads or triangles? Why do we provide both options?

Reply:
```suggestion
| **Import Quads** | The faces of a 3D GameObject are built of quads, and by default the ProBuilderize action maintains this build. Disable **Import Quads** to build the faces of triangles. |
```

Reply:
Please see the updated suggestion above.

Reply:
@suearkinunity Hmm, I would maybe not single out `animation` and just explain it more broadly. Something along the lines of:
`While quads are generally more practical to manipulate, the build process can result in topology that might not fully match the triangle topology of the original asset - this can lead to unexpected visual artifacts, deformation issues and so on. To build the faces of triangles, disable **Import Quads**.` .